build.gradle

```diff
@@ -47,7 +47,7 @@ ext {
     playServicesVersion = '16.1.0'
     firebaseMessagingVersion = '17.6.0'
     oss_licenses = '16.0.2'
-    kotlinVersion = '1.3.11'
+    kotlinVersion = '1.3.40'
```

Comment:
Since, we're updating kotlin version, let's make it most updated & stable version i.e 1.3.60?

Reply:
@geekanamika i have updated the Kotlin Version to last version